presto-spi/src/main/java/com/facebook/presto/spi/resourceGroups/SchedulingPolicy.java

```diff
@@ -17,5 +17,6 @@
 {
     FAIR,
     WEIGHTED,
+    WEIGHTED_SHARES,
```

Comment:
We should look into whether we can remove the other policies.

Reply:
Not sure about that, we made resource groups as non experimental in the last release. IMO these do need to be renamed - this is actually a fair scheduler with weights. We would then have: `FCFS, STOCHASTIC, FAIR_SHARE, QUERY_PRIORITY`.

Reply:
Well, I don't think we should be accumulating algorithms either. As we find ones that work better, we should be deprecating and removing the old ones.

Reply:
This new heuristic looks like a fair scheduler that takes into account the configured `schedulingWeight`, so how about naming it `WEIGHTED_FAIR`?

Reply:
That makes it sound related to 'FAIR', which it isn't at all..

Reply:
I still think `WEIGHTED_FAIR` is a better fit, but I leave the naming part to you guys to decide.